openstack/python/tar/bin/openstack-describe-instances

```diff
@@ -32,7 +32,8 @@ class OpenStackDescribeInstances(DescribeInstancesCommand, OpenStackCommand):
                  2: 'Terminated',
                  3: 'Pending',
                  4: 'Unknown',
-                 5: 'Stopped'}
+                 5: 'Stopped',
+                 7: 'Error'}
```

Comment:
Interesting. The link I provided point to `latest` so it should point to the latest version. 
Either they haven't updated the tag `latest` or they removed again these methods (`from/tostring()`).